webapp/components/admin_console/developer_settings.jsx

```diff
@@ -48,13 +48,13 @@ export default class DeveloperSettings extends AdminSettings {
                     label={
                         <FormattedMessage
                             id='admin.service.testingTitle'
-                            defaultMessage='Enable Testing: '
+                            defaultMessage='Enable Load Testing: '
```

Comment:
One other thing on this, we call the command `/loadtest`, but it really does more than that. Maybe we could even call it something like "Enable testing commands" to be more clear about it's purpose